Items/08_Tool_Sets.txt

```diff
@@ -0,0 +1,215 @@
+==============================
+Tool Sets
+==============================
+
+Blade Maintenance and Care Kit ($100):
+	A good whetstone, polish, oil and cleaning supplies are worth
+their weight in gold. These tools keep blades sharp, rust-free and
+ready for action.
+
+Relevant Skill(s): Weapons - Melee, Weapons - Thrown
+Context: Bladed Weapons
+Noise Level: Low
+Weight: 
+
+Digital Artist's Program and Tablet Package ($250):
+	A pressure-sensitive digital tablet and stylus complete with 
+TIR Digital Painting, which can be used for drawing, painting, and 
+editing photos and documents. Comes with a protective case and 
+rechargeable 8-hour battery.
+
+Relevant Skill(s): Performance, Deception
+Context: Digital Art, Imagery
+Noise Level: Low
+Weight:
+
+Destructive Entry Kit ($200):
```

Comment:
Should this replace the similar tools in the tool section of items?

Reply:
My endgame is to absorb that entire section into this document